### ISSUE TITLE
Next steps 5/4

### ISSUE THREAD

**Issue by oimartin** [dictybase-playground/learn-github-action#22]
1. Determine where values will come from to source command line arguments (some may be generative or default), make a manifest
2. Determine what the sender email should look like, as of now: Stock center robot<stockbot@mail.dictycr.org>
3. Create new html parser (adjust old) to parse current invoice/order markdown

**Comment by oimartin** [comment]
Argument | Type | Default | Get from |
-- | -- | -- | --
organization | default | dictyBase | default value?
repository | default | Stock-Center-Orders | default value?
issueid | Github action | NA | issue that triggered action, github.event.issue.number
label | Github action | NA | label that triggered action, github.event.issue.labels and iterate to get label?
endpoint | generative | - | input, hardcode string value?
apikey | generative | - | input, secrets.apikey?
token | generative| - | input, secrets.token?
sender | default | Stock center robot<stockbot@mail.dictycr.org> | default value?

possible label triggers: Growing / in Preparation and Shipped

**Comment by oimartin** [comment]
1. 
For default values, do we need to specify the value in the command line when calling main.py?

I started editing the workflow and merged it to the develop branch to see how it runs. I kept getting the following error about wheel package:
`Collecting wheel==0.33.6
  Downloading wheel-0.33.6-py2.py3-none-any.whl (21 kB)
Collecting wrapt==1.12.1
  Downloading wrapt-1.12.1.tar.gz (27 kB)
Collecting pycparser
  Downloading pycparser-2.20-py2.py3-none-any.whl (112 kB)
Could not build wheels for readline, since package 'wheel' is not installed.
Could not build wheels for wrapt, since package 'wheel' is not installed.`

I added the two labels we would use in the Stock-Center-Orders repo and the 'invalid' label from this repo, so I could test it in learn-github-action.
